packages/admin-theme/src/cometAdminOverrides/toolbar.ts

```diff
@@ -0,0 +1,15 @@
+import { CometAdminToolbarClassKeys } from "@comet/admin";
+import { StyleRules } from "@material-ui/styles/withStyles";
+
+export const cometAdminToolbarOverrides = (): StyleRules<{}, CometAdminToolbarClassKeys> => ({
+    root: {
+        minHeight: 80,
+    },
+    muiToolbar: {
+        "& [class*='MuiButton']": {
+            minHeight: 40,
+            alignItems: "center",
+        },
```

Comment:
This is not necessary, this will be the default styling of the MuiButton.

Reply:
i think the minHeight es special for the toolbar buttons

Reply:
removed it due to our diskussion

Reply:
All buttons heights are changed to 40px in #371